frontend/server/src/main/resources/log4j2.xml

```diff
@@ -77,9 +88,12 @@
 		<Logger name="TS_METRICS" level="all">
 			<AppenderRef ref="ts_metrics"/>
 		</Logger>
+		<Logger name="TELEMETRY_METRICS" level="all">
+				<AppenderRef ref="telemetry_metrics"/>
+		</Logger>
```

Comment:
it should be removed from default TorchServe log4j config. It will confuse community members. Nobody understand telemetry metrics